features/conversation/conversation-new-dm-no-messages-placeholder.tsx

```diff
@@ -1,58 +0,0 @@
-import { Center } from "@/design-system/Center";
```

Comment:
is this not needed any longer?

Reply:
I've put it inside conversation-new-dm for now since everything this file needed was also in that one and I felt it was more clear than having this one only external file